src/Settings.cpp

```diff
@@ -632,12 +661,15 @@ JackTrip *Settings::getConfiguredJackTrip()
         //netks->play();
         // -------------------------------------------------------------
     }
-    
+
     if (mIOStatTimeout > 0) {
         jackTrip->setIOStatTimeout(mIOStatTimeout);
         jackTrip->setIOStatStream(mIOStatStream);
     }
 
+    jackTrip->setAudioTesterP(&mAudioTester);
+
+    if (not mAudioTester.getEnabled()) { // No effects plugins allowed while testing:
     // Allocate audio effects in client, if any:
```

Comment:
fix indenting?

Reply:
Sure, I was just trying to minimize the diffs.  What I normally do is a minimized change like this followed by a separate commit flagged as "whitespace/formatting only", but given how hard it is to contribute commits in this system, I suppose it is better to go directly for one commit, even though it's more of a mess.

Reply:
On second thought, a pull request can technically contain any number of commits, so maybe the right process is to submit a multi-commit pull-request equivalent to the current result of a single-commit PR followed by a few update commits, designed for maximum clarity.